### ISSUE TITLE
Dialogues mixing together when using menu near an actor

### ISSUE THREAD

**Issue by Clovertea** [chrismaltby/gb-studio#468]
**Describe the bug**

Dialogues of actor and menu will mix together when using the menu near an actor, as if it is also interacting with the actor at the same time

**To Reproduce**
1. Add a new scene and an actor
2. in the 'On init' of the scene, add  "Joypad Input: Attach Script To Button", "Text: Display Menu", "Switch" and add some dialogues for the choices 
3. add different dialogues for the actor (On interact)
4. open the menu near the actor, pick any of the choices and you will see the dialogues mixing together

**Expected behavior**

The dialogues of the actor should not appear when using the menu

**Screenshots**
![gbs bug](https://user-images.githubusercontent.com/70198441/91153297-9dac2480-e6f2-11ea-881c-d37ab3e0006f.gif)

**Platform (please complete the following information):**
 - OS: Window 8.1
 - App version: gbs 2.0 beta

**Additional context**
project attached (just check Scene 16, the rest is as same as the sample project)
[sample gbs2.zip](https://github.com/chrismaltby/gb-studio/files/5122485/sample.gbs2.zip)

**Comment by pau-tomas** [comment]
I wonder if this and #457 are related. In both cases it seems like the user input is kept longer than usual triggering the next action assigned to that input (in this case interact with the other actor and in #457 closing the scene).

**Comment by RichardULZ** [comment]
Yeah, I expect the a button to select in menu is allowing you to interact immediately after the selection, without waiting for a release of the last joypad like previously.
